Cesium3DTilesSelection/include/Cesium3DTilesSelection/EllipsoidTilesetLoader.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <Cesium3DTilesSelection/Tile.h>
+#include <Cesium3DTilesSelection/TileLoadResult.h>
+#include <Cesium3DTilesSelection/Tileset.h>
+#include <Cesium3DTilesSelection/TilesetContentLoader.h>
+#include <Cesium3DTilesSelection/TilesetExternals.h>
+#include <Cesium3DTilesSelection/TilesetOptions.h>
+#include <CesiumAsync/Future.h>
+#include <CesiumGeometry/QuadtreeTileID.h>
+#include <CesiumGeometry/QuadtreeTilingScheme.h>
+#include <CesiumGeospatial/BoundingRegion.h>
+#include <CesiumGeospatial/Ellipsoid.h>
+#include <CesiumGeospatial/GeographicProjection.h>
+#include <CesiumGltf/Model.h>
+
+#include <glm/fwd.hpp>
+
+#include <cstdint>
+#include <memory>
+#include <vector>
+
+using namespace CesiumGltf;
+using namespace CesiumAsync;
+using namespace CesiumUtility;
+using namespace CesiumGeometry;
+using namespace CesiumGeospatial;
+
+namespace Cesium3DTilesSelection {
+class EllipsoidTilesetLoader : public TilesetContentLoader {
```

Comment:
It would be good to have some doc comments on this class and its members.

Reply:
I've added doc comments for public members, as I've usually seen done. Let me know if you meant to have these for private members as well.

Reply:
Thanks! No need for doc comments on private members.